### DISCUSSION TITLE
No TypeScript Support For `prefix` in `tailwind.config`?

### DISCUSSION THREAD

**Discussion by YTKme** [tailwindlabs/tailwindcss#12549] [Help] [answered]
Hello,

I am using Tailwind along with Vite for frontend. I wanted to add a `prefix` to the `tailwind.config.ts` file. But when I do, I get the below warning.

```
warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration
```

I left my `tailwind.config.ts` file mostly same except for 1 line

```
import type { Config } from 'tailwindcss'

const config: Config = {
  prefix: 'tw-',
  content: [
    './src/**/*.{js,ts,jsx,tsx,mdx}',
  ],
  theme: {
    extend: {
    },
  },
  plugins: [],
}

export default config
```

**Comment by wongjn** [comment]
Consider checking you have updated all your Tailwind classes to have the `prefix` when you set it in your configuration.

**Comment by alex-krasikau** [comment]
Hey @YTKme!

This warning tells that there're no files containing any tailwind classes in your `src` folder. As soon as you introduced the `tw-` prefix, you might want to update your classes to have this one.

**Comment by Zehad-10** [comment, marked as answer]
You are using `prefix: 'tw-',` so your classes should be eg. `tw-text-center` `tw-text-blue-600` add the prefix `tw-` before every class

**Comment by YTKme** [comment]
Apologies for the confusion, is the first time i am using tailwind, i was under the assumption that `prefix` will auto prefix all CSS classes with that given prefix. I think I got it now. Thank you for the help!
